app/views/comments/_index.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="add-comment">
+  <div class="comment-new">
+    <% if current_user.present? %>
+        <%= render 'comments/form' %>
+    <% else %>
+        <%= link_to 'Please sign in before comment the article', root_path %>
+    <% end %>
+  </div>
+</section>
+<section class='comments col-sm-12'>
+  <br>
+  <p>Comments:</p>
+  <% @comments.each do |comment| %>
+    <div class="comment">
+      <div class='row'>
+        <div class='content'>
```

Comment:
this class means that in this block contains comment content, but in [23 line](https://github.com/EvgeniiK/Mugenblog/pull/36/files#diff-76823f79ae20f1eeacbe9cd4cbe78362R22) there is `comment-content` class with the same meaning.